### PR TITLE
Us22 sign up page add method

### PR DESCRIPTION
**Pull request by AnnesophieMarchet**

### CODE REVIEW

File: server/app/routers/api/router.js
Comment (wilderLPM) [inline server/app/routers/api/router.js:1]: You should put "/user**s**"

File: client/src/components/Form/FormData.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+import { Form } from "react-router-dom";
+
+function FormData() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const [firstName, setFirstName] = useState("");
+  const [name, setName] = useState("");
+  const [birthDay, setBirthDay] = useState("");
+  const [sexe, setSexe] = useState("");
```
Comment (wilderLPM) [inline client/src/components/Form/FormData.jsx:11]: Should be "sex, setSex"

File: server/app/routers/api/users/router.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+// Define Your API Routes Here
+/* ************************************************************************* */
+
+// Import offer-related actions
+const { browse, signupUser } = require("../../../controllers/UserAction");
+
+// Route to get a list of offers
+router.get("/", browse);
+
+router.post("/signup", signupUser);
```
Comment (wilderLPM) [inline server/app/routers/api/users/router.js:15]: We need to check what the norm for routes and methods' names.
I would have put **router.post("/", add)**

File: server/database/models/UserRepositoty.js
Comment (wilderLPM) [inline server/database/models/UserRepositoty.js:1]: typo in the file name : UserReposito**r**y.js

**Review by wilderLPM — CHANGES_REQUESTED** [review]
A few small mistakes to fix but otherwise very good !

File: server/app/routers/api/router.js [hunk 3]
```diff
@@ -13,6 +13,10 @@ router.use("/offers", offersRouter);
 const companiesRouter = require("./companies/router");
 
 router.use("/companies", companiesRouter);
+
+const usersRouter = require("./users/router");
+
+router.use("/user", usersRouter);
```
Comment (wilderLPM) [inline server/app/routers/api/router.js:19]: You should put "/user**s**"

File: client/src/components/Form/FormData.jsx (hunk 1 above)
Comment (AnnesophieMarchet) [reply client/src/components/Form/FormData.jsx:11]: done

File: server/app/routers/api/router.js (hunk 3 above)
Comment (AnnesophieMarchet) [reply server/app/routers/api/router.js:19]: done

File: server/app/routers/api/users/router.js (hunk 2 above)
Comment (AnnesophieMarchet) [reply server/app/routers/api/users/router.js:15]: done

File: server/database/models/UserRepositoty.js
Comment (AnnesophieMarchet) [reply server/database/models/UserRepositoty.js:1]: done

**Review by wilderLPM — APPROVED** [review]
Great job !

**Pull request merged**